subsys/bluetooth/audio/shell/bap_broadcast_assistant.c

```diff
@@ -900,6 +903,23 @@ static int cmd_bap_broadcast_assistant_add_pa_sync(const struct shell *sh,
 		}
 	}
 
+	/* use the BASE to verify the BIS indexes set by command */
+	for (size_t j = 0U; j < param.num_subgroups; j++) {
+		if (bis_bitfield_req == 0) {
+			/* Request a PA sync without BIS sync */
+			subgroup_params[j].bis_sync = BT_BAP_BIS_SYNC_NO_PREF;
```

Comment:
`subgroup_params[j].bis_sync` should be `0`. `BT_BAP_BIS_SYNC_NO_PREF` indicated "No preference" and not "No sync"